contentcuration/contentcuration/frontend/settings/pages/Account/StudioCopyToken.vue

```diff
@@ -0,0 +1,138 @@
+<template>
+
+  <div class="token-input-wrapper">
+    <KCircularLoader
+      v-if="loading"
+      :shouldShow="loading"
```

Comment:
`shouldShow` is only for specific cases. Here it won't take effect since you use `v-if`. More [here](https://design-system.learningequality.org/kcircularloader#prop:shouldShow).